build.gradle

```diff
@@ -10,7 +10,8 @@ plugins {
 }
 
 group = "uk.nhs.tis.trainee"
-version = "0.20.0"
+
+version = "0.20.1"
```

Comment:
This functionality would be a minor patch per https://semver.org/
So, `0.21.0` would be more accurate.